scripts/deploy_docs.sh

```diff
@@ -36,6 +36,7 @@ if [ "$github_repo" = "leanprover-community/mathlib" -a "$github_event" = "push"
   git config user.email "leanprover.community@gmail.com"
   git config user.name "leanprover-community-bot"
   git add -A .
+  git reset --soft HEAD~1
```

Comment:
```suggestion
  git checkout --orphan master2
  git commit -m "automatic update to $git_hash"
  git push HEAD:master
```

This will discard the complete history.